src/utils.ts

```diff
@@ -198,7 +198,10 @@ export namespace Utils {
     }
 
     // verify base64
-    if (Buffer.from(operation.value.toString(), "base64").length !== 48) {
+    if (
+      !operation.value ||
+      Buffer.from(operation.value.toString(), "base64").length !== 48
+    ) {
```

Comment:
Nit: I think it would be more readable to move all undefined/null checks to the if statement (the one above) like:
```ts
    if (!operation.value) {
      throw new InvalidSep10ChallengeError(
          "The transaction's operation values should not be null",
      );
    }
```